### PR TITLE
install cron into the cli container

### PR DESCRIPTION
**Pull request by millidavids**

### CODE REVIEW

File: infrastructure/cli/Dockerfile [hunk 1]
```diff
@@ -10,8 +10,8 @@ ADD . /var/www/html
 
 WORKDIR /var/www/html
 
-# Install Composer dependencies
-RUN apt-get update && apt-get install -y git zip unzip \
+# Install dependencies
+RUN apt-get update && apt-get install -y git zip unzip cron \
```
Comment (bkuhl) [inline infrastructure/cli/Dockerfile]: I think we're using ubuntu as our base image.  This may already be on there

Comment (millidavids) [inline infrastructure/cli/Dockerfile]: Isn't cli built off of debian/jesse? I'm testing it right now.

Comment (bkuhl) [inline infrastructure/cli/Dockerfile]: You're right... it is.

This PR also needs to update the crontab - https://github.com/BibleBowl/account/blob/master/containers/cli/Dockerfile#L34

Comment (millidavids) [inline infrastructure/cli/Dockerfile]: I'm on it!

**Comment by jlaswell** [comment]
@millidavids Can we close this since we've got this updated to alpine?

**Comment by millidavids** [comment]
@jlaswell Yeah, we can close it , but there is still some cron configuration to do in the alpine images.

**Pull request closed without merging**